src/site/content/en/notifications/codelab-notifications-service-worker/index.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: codelab
+title: Use a Service Worker to manage notifications
+authors:
```

Comment:
Hey @katejeffreys,

The `authors` key has to be plural. Would you mind adding your information to `src/site/_data/contributors.json` ? Your username in that file should match the username you use in this doc. You'll also need to add a headshot to `src/site/content/en/images/authors/`. Please add a 1x and a 2x version. 1x is 96x96, 2x is 192x192.